lib/services/livesync-service.ts

```diff
@@ -92,7 +92,8 @@ export class LiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncServic
 				watchGlob: projectDir,
 				platformSpecificLiveSyncServices: platformSpecificLiveSyncServices,
 				notInstalledAppOnDeviceAction: notInstalledAppOnDeviceAction,
-				notRunningiOSSimulatorAction: () => Future.fromResult()
+				notRunningiOSSimulatorAction: () => Future.fromResult(),
+				getApplicationPathForiOSSimulatorAction:() => Future.fromResult("")
```

Comment:
Ping @Fatme - I'm really not sure what to pass here, I've added it as the code cannot be transpiled without this.